modules/dcache/src/main/java/diskCacheV111/services/PoolStatisticsV0.java

```diff
@@ -663,7 +610,7 @@ private void updateHtmlTop() throws IOException {
                 StringTokenizer st = new StringTokenizer(line);
 
                 st.nextToken();
-                String key =  _yearOfCalendar.get().format(new Date(Long.parseLong(st.nextToken())));
+                String key = new Date(Long.parseLong(st.nextToken())).toInstant().atZone(ZoneId.systemDefault()).format(_yearOfCalendar);
```

Comment:
yes ,i have also done simmilar like this
```
new Date(Long.parseLong(st.nextToken())).toInstant().atZone(ZoneId.systemDefault()).format(_yearOfCalendar);
```
but tigran suggest me this
```
_yearOfCalendar.format(Instant.ofEpochMilli(Long.parseLong(st.nextToken())))
```
which is't working